app/controllers/audio_files_controller.rb

```diff
@@ -6,12 +6,31 @@ def new
   def create
     @audio_file = current_user.audio_files.new(audio_file_params)
     if @audio_file.save
-      redirect_to dashboard_url, notice: "Audio file saved successfully"
+      Activity.create!(
```

Comment:
For readability purposes you may want to consider extracting a private method in responsible for creating the activity. Perhaps something like:

```ruby
if @audio_file.save
  record_activity(subject: @audio_file)
redirect_to ...
else
# ...
end
```

```ruby
def record_activity(subject:)
  Activity.create!(
    subject: subject,
    user: current_user,
  )
end
```